### PR TITLE
add tuning/config/configs unit tests

### LINKED ISSUE #72: Add unit tests for tuning/config/configs.py
## Is your feature request related to a problem? Please describe.

Add unit tests for the tuning/config/configs.py module.

## Describe the solution you'd like

If possible, each function should have test cases in the file tests/tuning/config/configs.py
and pass test with the command `pytest tests/tuning/config/configs.py`

## Additional context

Add any other context about the feature request here.

### PR DESCRIPTION
**Pull request by tedhtchang**
### Description of the change

Adds unit tests for tuning.config.configs modules

### Related issue number

Closes #72 

### How to verify the PR

```shell
git clone https://github.com/tedhtchang/fms-hf-tuning.git -b add-configs-unit-tests
cd ms-hf-tuning
pytest tests/configs/test_configs.py
```
### Was the PR tested

- [ ] I have added >=1 unit test(s) for every new method I have added.
- [x] I have ensured all unit tests pass

### CODE REVIEW

File: tests/configs/test_configs.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: Apache-2.0
```
[the remark this answers is not in the document]
Comment (tedhtchang) [reply tests/configs/test_configs.py:1]: This single line identifier is to avoid copy-paste block of the license @mrutkows ?

**Comment by tedhtchang** [comment]
@tharapalanivel I realized your PR#83 already test the configs module. I should close this PR unless you have examples of edges configs you want me to test.

File: tests/configs/test_configs.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import torch
+
+# Local
+import tuning.config.configs as tuning_config
+
+NEW_MODEL_NAME = "Maykeye/TinyLLama-v0"
+
+
+def test_model_argument_configs_default():
+    data_arguments = tuning_config.DataArguments
```
Comment (gkumbhat) [inline tests/configs/test_configs.py:25]: nit: can you add docstrings to these tests describing their goal and scope

**Comment by anhuong** [comment]
@tedhtchang can we close this PR if it has duplicative unit tests?

**Pull request closed without merging**